extension/page-detect.js

```diff
@@ -58,6 +60,8 @@ window.pageDetect = (() => {
 		return isRepo() && blobPattern.test(location.href);
 	};
 
+	const hasCommentForm = () => isGist() || (isRepo() && (isIssue() || isNewIssue() || (isPR() && !isPRFiles()) || isCompare()));
```

Comment:
Ok, removed the page checks. Since the `each`-loop won't run if there are no `.js-previewable-comment-form` elements, I just always run the `addUploadBtn` function.